src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/ia/CftQueryServiceClaimTaskTest.java

```diff
@@ -71,84 +64,6 @@ void setUp() {
         );
     }
 
-    @ParameterizedTest(name = "{0}")
-    @MethodSource("getGrantTypes")
-    void should_retrieve_a_task_to_claim(GrantType grantType) {
```

Comment:
1. Test data setup has commonality/duplicates for getTask(...), however the permission Types differ across the various actions (This one is Claim). This does not verify the scenario like assign "where it is a permission with MANAGE that will  action these tasks". Here we verify that only permission with OWN and EXECUTE can perform claim. Is this really a duplicate?
2. Looks like we have removed this because it's IA centric. Does that mean we have the WA equivalent?

Reply:
wa/CftQueryServiceClaimTaskTest
should_retrieve_a_task_when_grant_type_standard_
should_retrieve_a_task_when_grant_type_specific_